src/common/options/global.yaml.in

```diff
@@ -5365,6 +5365,30 @@ options:
   desc: Enable health indication when spurious read errors are observed by OSD
   default: true
   with_legacy: true
+- name: bluestore_slow_ops_warn_lifetime
+  type: uint
+  level: advanced
+  desc: Duration of slow op warning of bluestore should persist since last occurence
+  default: 30
```

Comment:
Shouldn't that be a much longer duration - e.g. an hour or even a day?
Overwise we might get a flapping alert which is easier to miss...